alerts/config.go

```diff
@@ -6,20 +6,26 @@ import (
 	"bytes"
 	"context"
 	"encoding/json"
+	"os"
+	"slices"
 	"time"
 
 	"github.com/tidepool-org/platform/data"
-	"github.com/tidepool-org/platform/data/blood/glucose"
+	nontypesglucose "github.com/tidepool-org/platform/data/blood/glucose"
```

Comment:
Rename this (and other occurrences) to `dataBloodGlucose` for consistency. I don't think `nontypesglucose` has any benefit over the existing alias used throughout the repository.